gengokit/httptransport/httptransport.go

```diff
@@ -101,6 +102,24 @@ func NewBinding(i int, meth *deftree.ServiceMethod) *Binding {
 		nField.LowCamelName = LowCamelName(nField.Name)
 
 		nBinding.Fields = append(nBinding.Fields, &nField)
+
+		// Emit warnings for certain cases
+		if !nField.IsBaseType {
+			log.Warnf("%s.%s is a custom type '%s', only base types and repeated base "+
+				"types are supported. As a result, the generated HTTP "+
+				"transport will fail to compile. Remove non-base types.",
+				meth.GetName(),
```

Comment:
Yes, that is what `gofmt` formatted it to look like. I think it relates to the first string literal not being on the same line as the opening parenthesis. I'll probably change that at some future point, but it's at least passing `gofmt` now.